### PR TITLE
Update to CirrusMDSDK 1.9.0

### PR DESCRIPTION
**Pull request by taylorjcase**
## Description
Updated sample code and documentation for version 1.9.0 of CirrusMDSDK.

## Motivation and Context
Version 1.9.0 of CirrsMDSDK added new logging options. You can now set logLevel on CirrusMDSDKSession to none or verbose. When set to verbose the SDK will print extensive logging to the debugger's console and the device's console for debugging purposes. The logLevel defaults to none.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have added tests to cover my changes.
- [x] No tests are required for this change.
- [x] My change requires a documentation change.
- [x] My change requires a CHANGELOG update.

### CODE REVIEW

**Comment by illyabusigin** [comment]
Nice work! Can I also request that this version be built with Xcode 11.4? Thanks!

**Comment by taylorjcase** [comment]
> Nice work! Can I also request that this version be built with Xcode 11.4? Thanks!

Absolutely! I am working on that as we speak. That will be version 1.9.1 and should be available in the next few days.

File: README.md [hunk 1]
```diff
@@ -499,7 +499,25 @@ CirrusMDChannel* channel = [CirrusMDSDKSession.singleton channels][index];
 [CirrusMDSDKSession.singleton navigateToChannelWithId: channel.id];
 ```
 
-### Additional API
+### Debugging
+
+#### Log Level
+
+The default log level is none. For debugging purposes this can be changed to verbose. When set to verbose the SDK will print extensive logging around network requests, network responses, state changes, and other useful information to the debugger's console and the device's console. When setting the log level to verbose for debugging purposes it should be done before calling anything other functions on the SDK.
```
Comment (DanielGalarza) [inline README.md:506]: I need to update the readme for Android too 👀

**Review by DanielGalarza — APPROVED** [review]
👍

**Pull request merged**